### PR TITLE
fix: create ghost build workflow

### LINKED ISSUE #1454: chore: investigate redundant build in pipeline
# Description

Find out why all the PRs want to execute `build`. But `build` is not there anymore. So it's just waiting for something that is never executed

### PR DESCRIPTION
**Pull request by aaa123eee**
# Description

This is a fix of the github's bug. Sometimes workflow isn't deleted after changes

https://github.com/orgs/community/discussions/26256

This PR adds an empty job just to have green CI

closes #1454

### CODE REVIEW

**Review by LayZeeDK — COMMENTED** [review]
This is not the way. We should be able to rename our workflow jobs. "build" has been renamed to "DTE Coordinator" (dte_coordinator).

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -191,3 +191,12 @@ jobs:
...
         uses: actions/deploy-pages@v1
+
+  # This is just a 'ghost' workflow to have all the github CI passed
+  build:
+    name: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: build
+        run: echo Hello, world!
```
Comment (hoebbelsB) [inline .github/workflows/build-and-test.yml:202]: whoot? :D can't we just re-name the DTE-Coordinator to build and remove the ghost one? ^^

Comment (LayZeeDK) [reply .github/workflows/build-and-test.yml:202]: I have left comments in [the issue](https://github.com/rx-angular/rx-angular/issues/1454).

**Comment by hoebbelsB** [comment]
hey @aaa123eee i'll close it as @LayZeeDK was completely right in saying it needs to be adjusted in the protected branch rules. Done that, fixed the issue

**Pull request closed without merging**